products/jbrowse-desktop/src/rootModel.ts

```diff
@@ -129,27 +127,25 @@ export default function rootModelFactory(pluginManager: PluginManager) {
           this.setSession(snapshot)
         }
       },
-      initializeInternetAccount(id: string, initialSnapshot = {}) {
-        const schema = pluginManager.pluggableConfigSchemaType(
-          'internet account',
-        ) as IAnyModelType
-        const configuration = resolveIdentifier(schema, self, id)
-
-        const accountType = pluginManager.getInternetAccountType(
-          configuration.type,
+      initializeInternetAccount(
+        internetAccountConfig: AnyConfigurationModel,
+        initialSnapshot = {},
```

Comment:
any reason for changing it to take a config instead of id?

Reply:
In the only place this is called, the calling code already has a handle the config. By passing it directly, we avoid having to call `resolveIdentifier` with the ID.